### PR TITLE
Limit RPC request sizes in HTTP transport layer

### PR DESCRIPTION
**Pull request by omerfirmak**

### CODE REVIEW

File: jsonrpc/http.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/NethermindEth/juno/utils"
 )
 
+const TEN_MB = 10 * 1024 * 1024
```
Comment (IronGauntlets) [inline jsonrpc/http.go:13]:
```suggestion
const MaxBytes = 10 * 1024 * 1024
```

**Review by IronGauntlets — APPROVED** [review]
What made you realise that we need to do this?

File: jsonrpc/http.go (hunk 1 above)
Comment (jelilat) [reply jsonrpc/http.go:13]:
```suggestion
const MaxBytes = 10 * 1024 * 1024 // 10MB
```

**Review by jelilat — APPROVED** [review]

**Comment by omerfirmak** [comment]
> What made you realise that we need to do this?

I was already aware for some time, also @wojciechos pointed it out.

**Pull request merged**